examples/react-apollo/cypress/support/e2e.ts

```diff
@@ -100,3 +113,26 @@ Cypress.Commands.add('disconnectBackend', () => {
 Cypress.Commands.add('getNavBar', () => {
   cy.get('.mantine-Navbar-root')
```

Comment:
Sorry, I didn't notice this in the previous PR.

Could you change this to `cy.findByRole('banner')` / `cy.findByRole('navigation')`? Provided that Mantine uses the proper semantic element for rendering the navigation bar.

There is [Testing Playground](https://chrome.google.com/webstore/detail/testing-playground/hejbmebodbijjdhflfknehhcgaklhano), which is a great tool for getting the most accessible queries.

Reply:
there are multiple `<nav>` elements (and none using the `banner` role), so I don't see other information than class to get the right one

Reply:
Then it'll be `navigation` and if you add an `aria-label` to the component describing what it is (e.g: `Main navigation`), then you'll be able to query it by using the following function call:

```ts
cy.findByRole(`navigation`, { name: /main navigation/i })
```

Relying on classes, especially of 3rd party UI libs can be dangerous if they change their implementation at some point in the future. It's a better practice to use accessible queries.